pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1,4 +1,4 @@
-{ pkgs }:
+{ stdenv, pkgs }:
```

Comment:
I don't know ... my feeling is that once you expect the entire frickin' package set as an argument, it's kind of silly to add other arguments on top of that. That's why I asked.